server/proxy/server.js

```diff
@@ -0,0 +1,90 @@
+const client = require('socket.io-client')
```

Comment:
This whole file has half lines with semicolons and half without, but I think Sisi was changing that on hers so you probably don't need to do it yourself